protocol/optyfi-protocol/strategy-composition.md

```diff
@@ -22,19 +22,18 @@ Depositing an underlying token into a pool as collateral and then borrowing an u
 
 ## Strategy Sequence
 
-A strategy sequence is comprised of multiple strategy steps. 
+A strategy sequence is composed of multiple strategy steps. 
```

Comment:
Same meaning, leave original as  'composed' is used more frequently.

Reply:
Changed to comprises as the use of 'comprised of' is incorrect.